packages/react-examples/src/react-menu/MenuList/MenuList.stories.tsx

```diff
@@ -78,16 +120,56 @@ export const MenuListWithRadios = () => {
   return (
     <Container>
       <MenuList checkedValues={checkedValues} onCheckedValueChange={onChange}>
-        <MenuItemRadio icon={<CutIcon />} name="checkbox" value="1" checkmark={checkmark}>
+        <MenuItemRadio icon={<CutIcon />} name="radio" value="1" checkmark={checkmark}>
           Item
         </MenuItemRadio>
-        <MenuItemRadio icon={<PasteIcon />} name="checkbox" value="2" checkmark={checkmark}>
+        <MenuItemRadio icon={<PasteIcon />} name="radio" value="2" checkmark={checkmark}>
           Item
         </MenuItemRadio>
-        <MenuItemRadio icon={<EditIcon />} name="checkbox" value="3" checkmark={checkmark}>
+        <MenuItemRadio icon={<EditIcon />} name="radio" value="3" checkmark={checkmark}>
```

Comment:
Can we avoid using `radio` as `name`? Let's use something more neutral as it might be confusing (`name=radio` vs `type=radio`)

Reply:
yeah i can do more realistic examples

Reply:
Updated the checkbox/radio examples to be more realistic